pgjdbc/src/test/java/org/postgresql/test/jdbc2/LoginTimeoutTest.java

```diff
@@ -81,8 +82,39 @@ public void testBadTimeout() throws Exception {
     props.setProperty("password", TestUtil.getPassword());
     props.setProperty("loginTimeout", "zzzz");
 
+    try {
+      DriverManager.getConnection(TestUtil.getURL(), props);
+      Assert.fail("bad loginTimeout should throw exception");
+    } catch (Exception e) {
+      // Ignore - expected
+    }
+  }
+
+  @Test
+  public void testUseDriverManagerLoginTimout() throws Exception {
+    Properties props = new Properties();
+    props.setProperty("user", TestUtil.getUser());
+    props.setProperty("password", TestUtil.getPassword());
+
+    // Case where it default to DriverManager.loginTimeout
+    DriverManager.setLoginTimeout(5);
+    Connection conn = DriverManager.getConnection(TestUtil.getURL(), props);
+    conn.close();
+    DriverManager.setLoginTimeout(0);
```

Comment:
Does this test actually test anything?
Would you please add some assertions?

Reply:
@zemian can you respond to this ?

Reply:
Hi Dave, as I was saying, I did replied on the Conversation tab regarding this (https://github.com/pgjdbc/pgjdbc/pull/900#issuecomment-341902369)... maybe it's not directly linked here. So let me re-paste it here for your reference:

> Hi @vlsi , I have put in a change that addressed your concerns now. Please review
>
>As far as test assertions. I have done similar to your existing LoginTimeoutTest, which contains few tests already doing similar thing (no assertion there), it merely ensure connection object are obtained correctly with timeout settings set. I doing same but with DriverManager.setLoginTimeout() instead. However I just now added one more new test for "testTimeoutOccursUsingDriverManagerLoginTimout", which also test just like existing one but with the DriverManager.setLoginTimeout(). This test has the assertion for real timeout. I think it's good validation for the change in this PR.

Reply:
or just remove this test?